sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
@crozeg after looking at this *very briefly*, I think perhaps the easiest and safest solution would probably just be to link against `libsortfilterproxymodel` directly in PROedit. Could you remove this new linker flag in this Qbs file, add the following to `src/app/avalon/avalon.qbs` and retest? I would expect this to work, but my system isn't affected, so I can't test it:

```
cpp.dynamicLibraries: ["sortfilterproxymodel"]
```